packages/template-retail-react-app/app/pages/product-list/partials/color-refinements.jsx

```diff
@@ -75,7 +84,16 @@ const ColorRefinements = ({filter, toggleFilter, selectedFilters}) => {
                                     alignItems="center"
                                     fontSize="sm"
                                     marginBottom="1px"
-                                >{`${value.label} (${value.hitCount})`}</Text>
+                                    aria-hidden="true" // avoid redundant readout since swatch has aria label
+                                >
+                                    {intl.formatMessage(
+                                        {
+                                            id: 'colorRefinements.label.hitCount',
+                                            defaultMessage: '{colorLabel} {colorHitCount}'
```

Comment:
```suggestion
                                            defaultMessage: '{colorLabel} ({colorHitCount})'
```
To match the previous text.